config/locales/es.yml

```diff
@@ -1,23 +1,188 @@
+        key_is: "Tu llave de API es %{key}."
+        credentials: "Si quieres usar %{gem_commands_link} desde la línea de comandos, vas a necesitar un archivo llamado %{gem_credentials_file}, el cual puedes generar usando el siguiente comando:"
+        link_text: "comandos de gem"
+        reset: "Reestablecer llave API"
+        confirm_reset: "¿Estás seguro? Este cambio no puede deshacerse."
+
+  rubygems:
+    dependencies:
+      header: "dependencias de %{title}"
+    edit:
+      title: "Editar enlaces"
+      subtitle: "para %{gem}"
+      other_fields_notice: "Los otros campos solo pueden ser editados al envíar una versión nueva de tu gema."
+    index:
+      title: "Gemas"
+      downloads: "Descargas"
+    show:
+      not_hosted_notice: "Esta gema no se encuentra alojada en RubyGems.org."
+      install: "instalar"
+      yanked_notice: "Esta gema ya no está disponible para descarga directa o para otras gemas que dependan de ella."
+      authors_header: "Autores"
+      downloads_for_this_version: "Para esta versión"
+      owners_header: "Administradores"
+      links:
+        header: "Enlaces"
+        home: "Sitio"
+        code: "Código de fuente"
+        docs: "Documentación"
+        wiki: "Wiki"
+        mail: "Lista de correo"
+        bugs: "Registro de bugs"
+        download: "Descargar"
+        subscribe: "Suscribirse"
+        report_abuse: "Reportar abuso"
+      versions_header: "Versiones"
+      show_all_versions: "Muestra todas las versiones (%{count} total)"
+      licenses_header: "Licencia"
+      no_licenses: "N/D"
+      requirements_header: "Requerimientos"
+      copy_to_clipboard: "Copiar al portapapeles"
+      copied: "Copiado!"
+      required_ruby_version: "Version de Ruby requerida"
+
+  searches:
+    show:
+      subtitle: "por %{query}"
+      exact_match: "Resultado exacto"
+
+  sessions:
+    new:
+      forgot_password: "¿Olvidaste tu contraseña?"
+
+  stats:
+    index:
+      total_downloads: "Descargas totales"
+      total_gems: "Gemas en total"
+      total_users: "Gemas en total"
+      all_time_most_downloaded: "Las Más Descargadas"
+
+  users:
+    new:
+      have_account: "¿Ya estás registrado?"
+
+  versions:
+    index:
+      title: "Todas las versiones de %{name}"
+      versions_since: "%{count} versiones desde %{since}"
+      not_hosted_notice: "Esta gema no está alojada en RubyGems.org."
+    version:
+      yanked: "eliminada"
+
   activerecord:
     attributes:
-      doorkeeper/application:
-        name: Nombre
-        redirect_uri:
+      user:
+        avatar: "Avatar"
+        email: "Dirección email"
+        handle: "Usuario"
+        password: "Contraseña"
       linkset:
-        bugs: URL del registro de Bugs
-        code: URL del código fuente
-        docs: URL de la documentación
-        mail: URL de la lista de correo
-        wiki: URL del Wiki
+        bugs: "URL del registro de Bugs"
+        code: "URL del código de fuente"
```

Comment:
Este me parece que si debe ser `código fuente`. `código de fuente` es como decir source of code. [https://es.wikipedia.org/wiki/Código_fuente](Código fuente) en wikipedia